src/main/webapp/app/entities/exercise-group.model.ts

```diff
@@ -5,7 +5,7 @@ import { BaseEntity } from 'app/shared/model/base-entity';
 export class ExerciseGroup implements BaseEntity {
     public id: number;
     public title: string;
-    public mandatory: boolean;
-    public exam: Exam;
-    public exercises: Exercise[];
+    public isMandatory: boolean;
+    public exam: Exam | null;
+    public exercises: Exercise[] | null;
```

Comment:
Is that a good design? Having the option of an has many relationship being null? Isn't an empty set well enough?